airgun/entities/host_new.py

```diff
@@ -311,6 +311,14 @@ def apply_erratas(self, entity_name, search):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
+    def get_errata_pagination(self, entity_name):
+        """Get pagination info from Errata tab on selected host"""
+        view = self.navigate_to(self, 'NewDetails', entity_name=entity_name)
+        view.content.errata.select()
+        view.wait_displayed()
+        view.content.errata.wait_displayed()
+        return view.read(widget_names="content.errata.pagination")
```

Comment:
I think the further issue is related to making a right locators for pagination. at this location https://github.com/SatelliteQE/airgun/blob/593616d3f3cbe06cd88aaa1f433c2aff0802f1b5/airgun/views/host_new.py#L362. I had chat with @vijaysawant which know the details about it.

Reply:
Yes he follow up with the new locator workaround i've committed here. 
Since there are two pagination objects on `newhost>content>errata` tab, 
we only read the top one, correct? Seeing success with this locally